src/redux/stats/statsSlice.js

```diff
@@ -0,0 +1,44 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const initialState = {
+  stats: [],
+  status: '',
+  error: null,
+};
+
+export const fetchStatsAsync = createAsyncThunk(
+  'stats/fetchStats',
+  async () => {
+    const response = await axios.get('https://financialmodelingprep.com/api/v3/income-statement/AAPL?limit=120&apikey=cd15d811a471e0cf71b9bbfac4de08c5');
+    return response.data;
+  },
+);
+
+const statsSlice = createSlice({
+  name: 'stats',
+  initialState,
+  reducers: {
+  },
+  extraReducers: (builder) => {
+    builder
+      .addCase(fetchStatsAsync.fulfilled, (state, action) => {
+        const newStats = [];
+        const keys = Object.keys(action.payload);
+        keys.forEach((keyOfActionPayload) => {
+          newStats.push({
+            id: keyOfActionPayload,
+            ...action.payload[keyOfActionPayload],
```

Comment:
- [ ] As per project requirements you are to use a `Filter stateless component `to `` data retrieved from the API. But I don't see you using the filter method anywhere.
![Screen Shot 2023-09-09 at 7 01 49 AM](https://github.com/MenendezON/react-capstone/assets/62849872/efd5b328-52b5-4993-93a5-aa04792d21ac)